### PR TITLE
docs: addition of param and result sections

### PR DESCRIPTION
**Pull request by hachreak**
- Improves documentation. (addresses #31)

Signed-off-by: Leonardo Rossi leonardo.r@cern.ch

### CODE REVIEW

File: invenio_jsonschemas/ext.py [hunk 1]
```diff
@@ -92,7 +93,10 @@ def get_schema_dir(self, path):
         """Retrieve the directory containing the given schema.
 
         :param path: schema path, relative to the directory when it was
```
Comment (alizeepace) [inline invenio_jsonschemas/ext.py]: where it was

File: invenio_jsonschemas/ext.py [hunk 2]
```diff
@@ -92,7 +93,10 @@ def get_schema_dir(self, path):
...
-        registered.
+            registered.
+        :raises invenio_jsonschemas.errors.JSONSchemaNotFound: If no schema
+            was found in the path speficied.
```
Comment (alizeepace) [inline invenio_jsonschemas/ext.py]: the specified path (also line 110, 123)

File: invenio_jsonschemas/jsonresolver.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
 
 @jsonresolver.hookimpl
 def jsonresolver_loader(url_map):
-    """JSON resolver plugin.
+    """JSON resolver plugin that load the schema endpoint.
```
Comment (alizeepace) [inline invenio_jsonschemas/jsonresolver.py]: loads

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio_jsonschemas/ext.py [hunk 4]
```diff
@@ -191,9 +217,9 @@ def init_app(self, app, entry_point_group=None):
 
     def init_config(self, app):
         """Initialize configuration."""
-        app.config.setdefault(InvenioJSONSchemas.CONFIG_ENDPOINT,
```
Comment (jirikuncar) [inline invenio_jsonschemas/ext.py:194]: please remove also `CONFIG_ENDPOINT = 'JSONSCHEMAS_ENDPOINT'`

Comment (hachreak) [reply invenio_jsonschemas/ext.py:194]: done

**Comment by hachreak** [comment]
ping @jirikuncar

**Comment by hachreak** [comment]
@jirikuncar  :shipit: ?

File: invenio_jsonschemas/views.py [hunk 5]
```diff
@@ -34,8 +34,8 @@
 def create_blueprint(state):
     """Create blueprint serving JSON schemas.
 
-    :param state: py:class:`invenio_jsonschemas.ext.InvenioJSONSchemasState`
-    instance used to retrieve the schemas
+    :param state: :class:`invenio_jsonschemas.ext.InvenioJSONSchemasState`
+       instance used to retrieve the schemas.
```
Comment (jirikuncar) [inline invenio_jsonschemas/views.py]: Can you check the indentation? (4 spaces)

**Review by jirikuncar — CHANGES_REQUESTED** [review]

File: invenio_jsonschemas/views.py (hunk 5 above)
Comment (hachreak) [reply invenio_jsonschemas/views.py]: done

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**